src/js/button.js

```diff
@@ -1,3 +1,8 @@
+import Component from './component';
+import * as Lib from './lib';
```

Comment:
I think we submitted our two comments at roughly the same time, so I never saw yours until a refresh.

There's some organizational overhead that I think we need to think through as well. If we break lib up, we're going to end up with a lot of extra noise in the root, and "browsers.js" doesn't really say anything about what it is. "utils/browsers.js" tells me that this is a group of utility functions that have something to do with browsers.